runtime/doc/lua-intro.txt

```diff
@@ -0,0 +1,107 @@
+*lua-intro.txt*	Nvim
+
+		       NVIM REFERENCE MANUAL
+
+			Introduction to Lua
+
+
+This chapter will give you the basics of using Lua in Neovim.
+
+                                      Type |gO| to see the table of contents.
+
+==============================================================================
+Introduction							     *lua-intro*
+
+COMMON QUESTIONS
+
+Maybe this section isn't needed. We'll see.
+
+- Is vimscript being deprecated?
+
+NEVER! (idk write something fancy)
+
+- Is my configuration gonna be faster?
+
+Probably not.
+
+- Explain difference between api vs lua api.
+
+Probably way more, blanking out at the moment.
+
+==============================================================================
+Using Lua from Neovim
+
+You can use `:lua` to execute a lua chunk. >
```

Comment:
Although lua uses the term "chunk", I'd maybe say "lua code" instead, for people unfamiliar with the language.